src/schema.ts

```diff
@@ -131,44 +164,44 @@ export function build(data: any, opt: QueryConfig = {}): Schema {
   return new Schema(fieldSchemas);
 }
 
-export class Schema {
+export class Schema implements TableSchema {
```

Comment:
Yeah, but it's kinda weird that we don't make them accessible.  

Now I start liking encapsulation more again.

If `tableSchema: ReadOnly<TableSchema<FieldSchema>>` is a property of Schema, then we can easily export the TableSchema object.  In addition, we can directly access all of the TableSchema via Schema?

Note that Schema's constructor is only called via the `build()` method, so we have the freedom to modify it however we like.